docs/fundamentals/document/document-api.md

```diff
@@ -102,6 +102,37 @@ da.get_attributes('tags__dimensions__height', 'tags__dimensions__weight')
 [[5.0, 5.0, 5.0, 5.0, 5.0, 5.0, 5.0, 5.0, 5.0, 5.0], [10.0, 10.0, 10.0, 10.0, 10.0, 10.0, 10.0, 10.0, 10.0, 10.0]]
 ```
 
+
+````{admonition} Note
+:class: caution
+
+As `tags` does not have a fixed schema, it is declared with type `google.protobuf.Struct` in the `DocumentProto`
+protobuf declaration. However, `google.protobuf.Struct` follows the JSON specification and does not 
+differentiate `int` from `float`. **So, data of type `int` in `tags` will be casted to `float` when request is
+sent to executor.**
+
+As a result, users need be explicit and cast the data to the expected type as follows.
+
+```{code-block} python
+---
+emphasize-lines: 7
```

Comment:
shouldn't be line 8 emphasized instead ?

Reply:
The line count starts from the first non-empty line, so in this case line 7 corresponds to this line
```index = int(doc.tags['index'])```
I think the above line needs to be emphasized because this where the data is casted.

If it makes things clearer, if I have also emphasized line 8: ``` assert self.animals[index] == 'dog'```.